keyserver/handler.go

```diff
@@ -56,3 +65,12 @@ func (server *ConiksServer) handleRegistrationMessage(reg *p.RegistrationRequest
 		TB:   tb,
 	}, nil
 }
+
+func (server *ConiksServer) StoreUserPoliciesToKV(up *p.ConiksUserPolicies) error {
```

Comment:
No problem, I've definitely forgotten to reply to comments too!

> Do you think that the server still needs to know the user's policies in some cases, e.g., *signed/unsigned key change* policy? If so, we have to store the user policies into db, right?

I think in the case of the lookup policy, I don't think the server needs to know. Depending on whether the server will verify key changes, then you're right, the server would still need to know the user's policy. In my suggestion, it's not that the user policies aren't stored in the DB, they're just made part of the blob of data mapped to the user's index and not stored separately. Do you think there would be too much overhead in retrieving the entire user leaf node from the DB to get something like a user policy bitmap or flag every time the server wants to check the key change policy?